mapping-for-navigation/_posts/0000-01-10-circular-junctions.md

```diff
@@ -0,0 +1,208 @@
+
+Roundabout is implied to be one way road but you may specify this explicitly by adding **[oneway](https://wiki.openstreetmap.org/wiki/Key:oneway)**=**[yes](https://wiki.openstreetmap.org/wiki/Tag:oneway%3Dyes)** tag. Additionally the number of **[lanes](https://wiki.openstreetmap.org/wiki/Key:lanes)**=**`*`** in the circle (typically 2, where long vehicles will need to use both) can also be tagged.
+
+Connect road lines from which the traffic flows to the roundabout. If a dual carriageway shares traffic with the roundabout, connect each its direction (one approaching, one departing) separately to the respective exit and entrance nodes on the roundabout. Don’t use the same node as entrance to and exit from a roundabout. Each road has to be connected with the roundabout in a separate node.
+
+When two roads share the same exit/entrance node connect them before the roundabout.
+
+[![][8c17feb2]][8c17feb2]
+
+  [8c17feb2]: ../../images/intersection-modeling/roundabouts/roundabout_pinch_ways.png
+
+**Roundabout shapes**
+
+Roundabouts are not necessarily circular in shape: there does exist roundabouts that are 0-shaped (oval) or D-shaped (with a straight segment, notably those created on a square whose one side is along an avenue).
+
+[![][c4d4f134]][c4d4f134]
+
+  [c4d4f134]: ../../images/intersection-modeling/roundabouts/roundabout_o_shape.png "O-shaped roundabout"
+
+See examples of roundabouts in the [OpenStreetMap Wiki](https://wiki.openstreetmap.org/wiki/Tag:junction%3Droundabout#Roundabouts).
+
+### Traffic circle
+
+A **traffic circle** is a type of intersection that directs both turning and through traffic onto a [one-way circular roadway](https://www.openstreetmap.org/user/daniel-j-h/diary/40101). Contrary to a roundabout, where entering traffic always yields to traffic already in the circle and merges in directly, the entrances to traffic circles are three-way intersections either controlled by stop signs, traffic signals, or not formally controlled.
+
+![][bceeea1b]
+
+  [bceeea1b]: ../../images/intersection-modeling/roundabouts/roundabout_traffic_circle.png "Traffic circle"
+
+The size of circular junctions and rotaries can vary from small to very large with several lanes, however generally **<u>the rotary or circular junction should not have any road network inside itself</u>**, this means circular ring roads around a city are not valid circular junctions.
+
+To designate such case of circular junction use tag **[junction][f14df547]**=**[circular][d11d94d4]**.
+
+  [d11d94d4]: https://wiki.openstreetmap.org/wiki/Tag:junction%3Dcircular
+
+**How to map**
+
+To add such a junction, use the same approach as for drawing roundabouts. Draw a circle line and tag it accordingly to the class of the roads that join it. Mark the appropriate nodes with traffic lights and traffic signs that control the priority of the movement of traffic flows.
```

Comment:
> Draw a circle line in the direction of the flow of traffic

Should that be clarified as it is in the roundabouts instructions?

Reply:
I've added anchor to link this to **how to** section of roundabout

```
**<a id="how-to-map-roundabout"></a>How to map**

To indicate a roundabout — draw a closed line in the direction the traffic flows,
…
To add such a junction, use the same approach as for [drawing roundabouts](#how-to-map-roundabout).
```